packages/api/src/task/task.ts

```diff
@@ -84,21 +88,65 @@ export default class TaskScheduler {
             status: "ready",
           });
 
-          await db.task.update(task.id, {
-            status: {
-              phase: "completed",
-              updatedAt: Date.now(),
-            },
-            output: event.output,
-          });
           return true;
```

Comment:
Transcode task also generates an output asset. We need a similar logic for it here. (could be related to the confusing `originTaskId` as well)